src/test/rpc/RPCCall_test.cpp

```diff
@@ -6216,9 +6216,46 @@ class RPCCall_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testSubmitResponseFields()
```

Comment:
Do I need to modify this unit test to incorporate apiVersion ? Since I'm adding new fields, it shouldn't break backwards compatibility even if an older version of rippled recieves the new output

Reply:
Correct, this should not be a breaking change and therefore should not be on the next apiVersion.